### PR TITLE
Desktop version Pull Request

### PR DESCRIPTION
**Pull request by tashisrael**
This pull request consists of the desktop version of my personal portfolio with media queries

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -6,16 +6,24 @@
...
+			<nav class="desktop">
+			<div class="logo">
 				<a href=""><p>My Logo</p></a>
-			</nav>
-			<nav class="menuitems">
+			</div>
+			<div class="desk-item">
+	          <div><a href="#portfolio">Portfolio</a></div>
+	          <div><a href="#about">About</a></div>
+	          <div><a href="#contact">Contact</a></div>
```
Comment (thecodechaser) [inline index.html:22]: - [ ] Please increase the font weight for the menu options and also increase the gap between them to match the [Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) design.

### Your design
image

### Figma design
image

File: index.html [hunk 2]
```diff
@@ -6,16 +6,24 @@
...
+            </div>
```
Comment (thecodechaser) [inline index.html:23]: - [ ] Please increase the font weight for all headings on the webpage and make sure it matches the Figma design.

### Your design
image

### Figma design
image

File: index.html [hunk 3]
```diff
@@ -56,9 +65,30 @@ <h2 class="tonic">Tonic</h2>
...
+				  <div class="grid-item32"><p>Full Stack Dev</p></div>
+				  <div class="grid-item42"><p>.</p></div>
+				  <div class="grid-item52"><p>2015</p></div>
+				</div>
+				<p class="day2">Experimental content creation feature that allows users to add to an existing story over the course of a day without spamming their friends.</p>
+				<ul class="langs2">
+					<li class="btn12">html</li>
+					<li class="btn52">Ruby on rails</li>
+					<li class="btn22">css</li>
+					<li class="btn32">javaScript</li>
```
Comment (thecodechaser) [inline index.html]: - [ ] Please decrease the gap between the tags and make sure it matches the Figma design.

### Your design
image

### Figma design
image

File: index.html [hunk 4]
```diff
@@ -110,6 +182,7 @@ <h2 class="tonic4">Multi-Post Stories</h2>
 					<li class="btn34">javaScript</li>
 				</ul>
 				<button class="btn44" type="submit">See Project</button>
+				</div>
 			</div>
 		</section>
```
Comment (thecodechaser) [inline index.html:187]: - [ ] Please decrease the form fields width size and make sure it matches the Design.

### Your design
image

### Figma design
image

File: index.html [hunk 5]
```diff
@@ -110,6 +182,7 @@ <h2 class="tonic4">Multi-Post Stories</h2>
 					<li class="btn34">javaScript</li>
 				</ul>
 				<button class="btn44" type="submit">See Project</button>
```
Comment (thecodechaser) [inline index.html:184]: - [ ] Please implement the interactions for all the buttons. (enabled, pressed, etc).

image

**Review by thecodechaser — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ 

## Hi @tashisrael👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 🎉
- No linter errors. :heavy_check_mark:
- GitHub flow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and summary. ✔️



## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 6]
```diff
@@ -178,10 +251,10 @@ <h2>Contact me</h2>
         <form action="https://formspree.io/f/mwkzpvlw" id="form" method="post">
```
Comment (Ismailco) [inline index.html:251]: - [ ] [OPTIONAL] Good job decreasing the form fields width size to make it match the Design, however, the form is not aligned properly. kindly align it so it is in the center of the page like in the Figma design

**Review by Ismailco — APPROVED** [review]
### Hi @tashisrael, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

cr-approve


## Highlights

✔️  No linter errors
✔️ Used Github Flow
✔️ Work documented professionally

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 6 above)
Comment (tashisrael) [reply index.html:251]: I have done that. Thank you

**Pull request merged**